backends/gdx-backend-robovm-metalangle/src/com/badlogic/gdx/backends/iosrobovm/IOSApplicationConfiguration.java

```diff
@@ -51,6 +51,9 @@ public class IOSApplicationConfiguration {
 	/** whether to use the compass, default true * */
 	public boolean useCompass = true;
 
+	/** whether to use the haptics engine, default false. * */
```

Comment:
It's set to `false` by default for performance reasons, to avoid creating and starting the `CHHapticEngine`. I'm not sure what you would like to be added to the docs, nothing is used when set to `false`, just the haptics calls won't do anything. It follows the same idea as `useCompass` or `useAccelerometer`

Reply:
Worth adding I haven't measured the performance impact and that it could be a better choice to just enable it by default if that impact is negligible but it's that easy to test. Because this would affect all apps, I applied a caution principle.

Reply:
Ah, sorry. I have understood it in the way that if it is set to `false`, old behaviour is used. Okay, it makes sense this way then.